frontend/src/pages/frontoffice/SignUp.js

```diff
@@ -46,11 +67,154 @@ export default function SignUp() {
               fontWeight: "bold",
               fontSize: "36px",
             }}
+            display={{ xs: "none", sm: "block" }}
           >
             Crear cuenta
           </Typography>
-          {/* <Step1 /> */}
-          <Step2 />
+          <Grid container pb={2} display={{ xs: "flex", sm: "none" }}>
+            <Grid item xs={3}>
+              <IconButton size="small" onClick={() => navigate("/sign-in")}>
+                <KeyboardArrowLeft />
+              </IconButton>
+            </Grid>
+            <Grid
+              item
+              xs={6}
+              sx={{
+                alignSelf: "center",
+                displey: "flex",
+              }}
+            >
+              <Typography
+                sx={{
+                  textAlign: "center",
+                  display: "flex",
+                  justifyContent: "center ",
+                  fontWeight: "bold",
+                }}
+              >
+                Crear cuenta
+              </Typography>
+            </Grid>
+          </Grid>
+          {/* <Box sx={{ border: "1px solid black", p: 2, borderRadius: 5 }}> */}
+          <Box pt={5}>
+            <Box
+              component="form"
+              noValidate
+              onSubmit={handleSubmit(onSubmit)}
```

Comment:
aca no seria lo mismo mandar el onSubmit de una?
```suggestion
              onSubmit={onSubmit}
```

Reply:
Tiene que ser por el react-hook-form